.github/workflows/update-deb-package-snapshots.yml

```diff
@@ -26,7 +31,7 @@ jobs:
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@38e0b6e68b4c852a5500a94740f0e535e0d7ba54 # v4.2.4
       with:
-        token: ${{ secrets.ACTIONS_TOKEN }}
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
ACTIONS_TOKEN is required here, this allows the generated PR to have actions triggered against it.

Reply:
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow

although I think I should re-make ACTIONS_TOKEN to be repo scoped